components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/Utils.java

```
@@ -52,9 +56,21 @@
             log.error("Error occurred while processing query parameters", e);
```

Should we silently ignore this error?